src/Migrations/Version20191124011509.php

```diff
@@ -10,7 +10,7 @@
 /**
  * Auto-generated Migration: Please modify to your needs!
  */
-final class Version20191116210807 extends AbstractMigration
+final class Version20191124011509 extends AbstractMigration
```

Comment:
Reikėtų mokintis pridėti tik savo pakeitimų migracijas.

T.y.
* Švarus projektas: `bin/console d:m:m --force`
* Padarai savo pakeitimą, sugeneruoji naujus: `bin/console d:m:d`
* Sudedi į GitHub'ą

Tada naujam žmogui (ar ir serveriui): nesipjaus versijos ir užteks `bin/console d:m:m`

> P.S. `d:m:m` yra santrumpos iš https://symfony.com/doc/master/bundles/DoctrineMigrationsBundle/index.html#usage

Reply:
Kodas veikia :heavy_plus_sign: buvo atskira pasakita (gal ir be namų darbų) apie `Doctrine` :heavy_plus_sign: vėlai atrašau (jau greitai final atsiskaitymas) – tai už namų darbus balo nemažinu – palieku tavo sąžinei, ar verta pasigilinti